salt/modules/virt.py

```diff
@@ -3946,44 +3935,66 @@ def migrate_non_shared(vm_, target, ssh=False, **kwargs):
 
 def migrate_non_shared_inc(vm_, target, ssh=False, **kwargs):
     """
-    Attempt to execute non-shared storage "inc" migration
+    .. deprecated:: 3002 Use ``virt.migrate`` with ``copy_storage='inc'`` instead.
```

Comment:
Same here, but wait... this was already pushed this way? better check with release managers what they think of it.